homeassistant/components/timer/strings.json

```diff
@@ -29,5 +29,18 @@
         }
       }
     }
+  },
+  "issues": {
+    "deprecated_duration_in_start": {
+      "title": "The timer start service duration parameter is being removed",
+      "fix_flow": {
+        "step": {
+          "confirm": {
+            "title": "The timer start service duration parameter is being removed",
+            "description": "The timer service `timer.start` optional duration parameter is being removed and use of it has been detected.\n\nPlease remove the use of this service from your automations and scripts and select **submit** to close this issue."
```

Comment:
If they are using the duration parameter, what should they do instead?

Reply:
You mean we should tell them they need to create a new timer?